### PR TITLE
Classical risk test

### PR DESCRIPTION
**Pull request by micheles**
Closes https://github.com/gem/oq-risklib/issues/429.

### CODE REVIEW

File: openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-001.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
...370506E-17 5.2043177E-17 4.4886152E-17 3.8731951E-17 3.3437542E-17 2.9536724E-17 2.6099836E-17 2.3070602E-17 2.0399783E-17 1.8044098E-17 1.6265762E-17 1.4666164E-17 1.3226948E-17 1.1931767E-17 1.0765883E-17 9.8675037E-18 9.0455633E-18 8.2934973E-18 7.6052086E-18 6.9752038E-18 6.4811958E-18 6.0228278E-18 5.5976218E-18 5.2029644E-18 4.8367794E-18 4.5452917E-18 4.2716910E-18 4.0149730E-18 3.7739983E-18 3.5477619E-18 3.3654062E-18 3.1925638E-18 3.0288322E-18 2.8736757E-18 2.7266255E-18 2.6067747E-18 2.4922832E-18 2.3828832E-18 2.2784407E-18 2.1786876E-18 2.0965540E-18 2.0176360E-18 1.9417327E-18 1.8687100E-18 1.7985010E-18 1.7402838E-18 1.6839425E-18 1.6294770E-18 1.5768203E-18 1.5258385E-18 1.4832977E-18 1.4418959E-18 1.4016999E-18 1.3626429E-18 1.3247247E-18 1.2928358E-18 1.2616840E-18 1.2312691E-18 1.2016581E-18 1.1727840E-18 1.1482644E-18 1.1243478E-18 1.1009002E-18 1.0779885E-18 1.0555457E-18 1.0364527E-18 1.0176946E-18 9.9927140E-19 9.8118324E-19 9.6349698E-19 9.4828957E-19 9.3341707E-19 9.1867858E-19 9.0427498E-19 8.9007240E-19 8.7781265E-19 8.6582086E-19 8.5396307E-19 8.4223923E-19 8.3071640E-19 7.4650584E-19 6.7107147E-19 6.0334130E-19 5.4264537E-19 4.8811287E-19,4.5017599E+02
```
Comment (ptormene) [inline openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-001.csv]: Please sort by asset

Comment (micheles) [inline openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-001.csv]: I have fixed this in the next poll request: https://github.com/gem/oq-risklib/pull/553

File: openquake/calculators/event_based_risk.py [hunk 2]
```diff
@@ -59,14 +59,21 @@ def build_agg_curve(r_data, insured_losses, ses_ratio, curve_resolution,
...
-            for i in range(insured_losses + 1):
-                # if insured_losses is False, don't build them
+            losses, poes = scientific.event_based(
+                data['loss'][:, l, 0], ses_ratio, curve_resolution)
+            avg = scientific.average_loss((losses, poes))
+            result[l, r, 'losses'] = losses
+            result[l, r, 'poes'] = poes
+            result[l, r, 'avg'] = avg
+            if insured_losses:
                 losses, poes = scientific.event_based(
```
Comment (ptormene) [inline openquake/calculators/event_based_risk.py]: please rename `losses`, `poes` and `avg` into `losses_ins`, `poes_ins` and `avg_ins`

Comment (micheles) [inline openquake/calculators/event_based_risk.py]: Done

**Comment by ptormene** [comment]
Please resolve conflicts. Apart from that, LGTM

**Pull request merged**